### PR TITLE
[16.0][MIG] partner_company_type: Migrate to version 16.0

### PR DESCRIPTION
**Pull request by apineux**
#1353

### CODE REVIEW

File: partner_company_type/README.rst [hunk 1]
```diff
@@ -14,16 +14,16 @@ Partner Company Type
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
-    :target: https://github.com/OCA/partner-contact/tree/15.0/partner_company_type
+    :target: https://github.com/OCA/partner-contact/tree/16.0/partner_company_type
     :alt: OCA/partner-contact
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
     :target: https://translation.odoo-community.org/projects/partner-contact-15-0/partner-contact-15-0-partner_company_type
```
Comment (baimont) [inline partner_company_type/README.rst:20]: :target: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_company_type

File: partner_company_type/readme/CREDITS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This module is maintained by:
+
+* Odoo Community Association
```
Comment (emagdalenaC2i) [inline partner_company_type/readme/CREDITS.rst:3]: I think this file could be deleted

**Review by emagdalenaC2i — APPROVED** [review]
LGTM, Just a little comment about the Credits.rst file

File: partner_company_type/README.rst (hunk 1 above)
Comment (legalsylvain) [reply partner_company_type/README.rst:20]: No need to change. Bot will do

**Review by legalsylvain — APPROVED** [review]

**Comment by marielejeune** [comment]
@rousseldenis

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**